smart-contracts/truffle-config.js

```diff
@@ -56,12 +62,10 @@ module.exports = {
           enabled: true,
           runs: 200,
         },
-        evmVersion: 'petersburg',
       },
     },
   },
   mocha: {
     useColors: true,
   },
-  plugins: ['truffle-security'],
```

Comment:
Why remove this?  It should have 0 impact.  It enables the `npx truffle verify` command

Reply:
same here